src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -7505,6 +7505,11 @@ static void writeStaffDetails(XmlWriter& xml, const Part* part)
                     xml.endElement();
                 }
             }
+
+            if (st->staffMag(Fraction(0, 1)) != 1.0) {
```

Comment:
It's better to call staffMag only once (add a temp variable and reuse it)

Reply:
Compararing double values for (non-)equality??

Reply:
@Jojo-Schmitz Yes, why not? It's not critical here to watch out for rounding issues.

Reply:
@Jojo-Schmitz is right. You should use RealIsEqual here and elsewhere. We should always use the functions from realfn.h to compare floating numbers instead of using == and !=, as they are much safer

Reply:
@RomanPudashkin thanks. Done.